ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```
@@ -429,7 +429,10 @@ objects:
         fi
         if [[ "${CLUSTER_TYPE}" = "aws" ]]; then
             master_type=null
-            if [[ "${CLUSTER_VARIANT}" =~ "xlarge" ]]; then
+            worker_type=m4.xlarge
+            if [[ "${CLUSTER_VARIANT}" == "xlarge_worker" ]]; then
```

```suggestion
            if [[ "${CLUSTER_VARIANT}" =~ "xlarge_worker" ]]; then
```

i don't know if there's mutation/manipulation of the string that might cause the exact match to fail.